compose.yaml

```diff
@@ -25,5 +25,6 @@ services:
       PGUSER: *DB_USERNAME
       PGPASSWORD: *DB_PASSWORD
       PGHOST: postgres
+      CLIENT_URL: http://68.219.109.240:5000
```

Comment:
Don't change, it should be changed only on VPS manually

Reply:
Add a variable?